.github/workflows/filter-frontend.yml

```diff
@@ -1,12 +0,0 @@
-name: filter-frontend
-on:
-  pull_request:
-  merge_group:
-    paths-ignore:
-      - '**.js'
-      - '**/package.json'
```

Comment:
Instead of removing this filter action entirely, would it make sense to add the `paths-ignore` to `pull_request` here?